services/web/server/tests/unit/with_dbs/03/wallets/test_wallets.py

```diff
@@ -29,8 +29,10 @@
 def mock_rut_sum_total_available_credits_in_the_wallet(
     mocker: MockerFixture,
 ) -> mock.Mock:
+    # NOTE: PC->MD should rather use aioresponse to mock RUT responses
```

Comment:
Sure you want to keep this note?

Reply:
I am ok with it, it reminds me that I should improve the mocking mechanism there.